solr/core/src/test/org/apache/solr/cloud/api/collections/SimpleCollectionCreateDeleteTest.java

```diff
@@ -104,6 +109,29 @@ public void testCreateAndDeleteThenCreateAgain() throws Exception {
     }
   }
 
+  @Test
+  @SuppressWarnings("unchecked")
```

Comment:
Would be good to add the test where it verifies all the expected properties. If test sees new property then it fails. So that person can justify any change in future.

Reply:
Yeah, I think it is a good idea